instance/management/commands/instance_statistics_csv.py

```diff
@@ -75,6 +75,12 @@ def add_arguments(self, parser):
             help='Comma-separated list of fully qualified domains of the instances to get the data for',
             required=True
         )
+        parser.add_argument(
+            '--users',
+            '-u',
+            action='store_true',
+            help='Gather active user statistics for all instances',
+        )
```

Comment:
@janimo, can we enable this for all runs of this command instead of gating it behind a flag?